backend/tests/test_location_implementation.py

```
@@ -0,0 +1,116 @@
+bedroom = Location.objects.create(user=user, display_name="Bedroom", parent=home)
+print(f"Created: {bedroom.full_path} (level {bedroom.level})")
+
+closet = Location.objects.create(user=user, display_name="Closet", parent=bedroom)
+print(f"Created: {closet.full_path} (level {closet.level})")
+
+garage = Location.objects.create(user=user, display_name="Garage", parent=home)
+print(f"Created: {garage.full_path} (level {garage.level})")
+
+print("\n=== 2. Full Path Materialization ===")
+print(f"Closet full path: {closet.full_path}")
+print(f"Expected: home/bedroom/closet")
```

Ruff (0.15.2)

[error] f-string without any placeholders

Remove extraneous `f` prefix

(F541)